product/views.py

```diff
@@ -1,3 +1,18 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.views import View
+from django.http  import JsonResponse
+
+from product.models import Category, SubCategory
+
+
+class CategoryView(View):
+    def get(self, request):
+        categories = Category.objects.all()
+
+        result = [{'id': category.id,
+                   'category' : category.name,
+                   'subcategories': sub_category.name} for category in categories
+                   for sub_category in SubCategory.objects.filter(category=category)]
```

Comment:
```suggestion
        result = [
            {
                'id': category.id,
                'category' : category.name,
                'subcategory': [
                    {
                        "sub_category_id     : sub_category.id,
                        "sub_category_name : sub_category.name
                    } for sub_category in SubCategory.objects.filter(category=category)
                ]
                } for category in categories ]
```
이렇게 하시면 될 것 같습니다. 근데 꼭 카테고리 뿌릴때 서브 카테고리도 같이 뿌려야 하나요?